cmd/test_load.go

```diff
@@ -202,3 +224,31 @@ func (lt *loadedTest) consolidateDeriveAndValidateConfig(
 
 	return nil
 }
+
+type syncWriter struct {
+	w io.Writer
+	m sync.Mutex
+}
+
+func (cw *syncWriter) Write(b []byte) (int, error) {
+	cw.m.Lock()
+	defer cw.m.Unlock()
+	return cw.w.Write(b)
+}
+
+func (lt *loadedTest) createFileForAppend(filename string) (io.WriteCloser, error) {
+	fs := lt.fileSystems["file"]
+	if ok, err := afero.Exists(fs, filename); !ok {
+		// This whole parts is because afero has problems creating files with OpenFile.
```

Comment:
```suggestion
		// This whole part is because afero has problems creating files with OpenFile.
```

Reply:
This seems to be because I was using `filesystems` ... hopefully this will fix the windows failing tests as well :crossed_fingers: